python/tvm/relay/frontend/common.py

```diff
@@ -563,6 +563,14 @@ def infer_value_simulated(input_val, params):
     return output_value
 
 
+def try_infer_value(val, on_success, on_failure):
```

Comment:
Can we make these callback to be optional, and just return original value of fail?

Reply:
Thanks for the suggestion, I also like this change. Updated.